### PR TITLE
Update pairs.py

### PR DESCRIPTION
**Pull request by ash487**
Greetings Quantopian Community,

I was at the NYC Event on Pairs Trading, and the current example algorithm is deprecated, such that one cannot deploy it in live trading. With this fix, users can now deploy the algorithm in live trading.

### CODE REVIEW

File: lectures/pairs_trading/pairs.py [hunk 1]
```diff
@@ -9,25 +9,24 @@ def initialize(context):
     # Quantopian backtester specific variables
     set_slippage(slippage.FixedSlippage(spread=0))
     set_commission(commission.PerTrade(cost=1))
-    set_symbol_lookup_date('2014-01-01')
-    
-    context.stock_pairs = [(symbol('ABGB'), symbol('FSLR')),
-                           (symbol('CSUN'), symbol('ASTI'))]
+    set_symbol_lookup_date('2010-01-01')
+
+    context.stock_pairs = [(symbol(' '), symbol(' '))]
```
Comment (CaptainKanuk) [inline lectures/pairs_trading/pairs.py]: It seems the symbols and lookup date were inadvertently removed here.

File: lectures/pairs_trading/pairs.py [hunk 2]
```diff
@@ -36,17 +35,21 @@ def handle_data(context, data):
...
-    
+
     for i in range(context.num_pairs):
 
         (stock_y, stock_x) = context.stock_pairs[i]
 
-        Y = prices[stock_y]
-        X = prices[stock_x]
+
+       ***THE CHANGE***
```
Comment (CaptainKanuk) [inline lectures/pairs_trading/pairs.py]: This change is good, could we remove the comments indicating that it has been changed?

Comment (ash487) [inline lectures/pairs_trading/pairs.py]: Sure, do I need to remove the comments, or can that be done on your end?

**Comment by CaptainKanuk** [comment]
Hey Ash, would you have time to make these changes? If not I'll try to do it in the next couple weeks.

**Comment by mmargenot** [comment]
I changed the algo to use the new optimization API so I am closing this.

**Pull request closed without merging**